packages/cactus-common/src/main/typescript/socketio-test-setup-helpers.ts

```diff
@@ -2,24 +2,23 @@
  * Helper module for setting up client/server test sockets.
  */
 
-import ioServer from "socket.io";
-import io from "socket.io-client";
+import { Server, Socket as ServerSocket } from "socket.io";
+import { io, Socket as ClientSocket } from "socket.io-client";
 import { createServer } from "http";
 
-export { ioServer };
+export { Server, ServerSocket, ClientSocket };
 
 /**
```

Comment:
@outSH Apologies for the nit picking: Could you move this file to the `cactus-test-tooling` package instead of `cactus-common`? 
Reasoning: If it's specifically meant for testing then I'd rather not have it increase the bundle size of the common package. I know that the `Servers` utility class looks similarly out of place, but that one was originally meant to be the utility that the `cmd-api-server` uses when doing graceful HTTP server shutdowns so that's why that one had to be put here in the `cactus-common` package. The fact that 99% of its use-cases for the `Servers` class is actually in the test automation is just a happy coincidence.

Reply:
@petermetz Sure, no problem, any tips are welcomed :) I was in fact mimicking `Servers` class and thus put it in the same folder, but I agree with your reasoning. I've moved `socketio-test-setup-helpers` to `cactus-test-tooling` both in this PR, and in #1660 (from which this change originated).